MACILE-guide.md

```diff
@@ -0,0 +1,42 @@
+# MACILE's guide
+
+### Logo
+
+![logo-macile](images/logo-macile.jpg)
+
+### Tipografía
+
+1. `Trajan Pro Bold`
+
+2. `Trajan Pro Regular` 
+
+### Colores
+
+1. **Azul**
+
+![azul](images/azul.png)
```

Comment:
We can add a bit more context to this image files. It is mostly for a maintainer. When I go look at the image folder, I don't know what blue, gold, yellow files are. If i add a modifier, such as `swatch-` or `color-swatch-` in front of each one, I know what each is without having to open the file